### DISCUSSION TITLE
OneToMany typescript dont see

### DISCUSSION THREAD

**Discussion by productdevbook** [mikro-orm/mikro-orm#3155] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/38668796/170625690-1ae13626-1ef7-4e82-932c-4f71013ae2fa.png)

dont working ts, mappedBy entity etc dont see. How to working ?

**Comment by B4nan** [comment]
You need to start providing reproductions if you need help.

**Comment by productdevbook** [comment, marked as answer]
When both files appeared after typing the name, ts values were visible.
